docs/docs/rest-api/public/api/v2/apps.raml

```diff
@@ -0,0 +1,451 @@
+
+###
+# Get a list of applications
+#
+get:
+  description: Get the list of running applications. Several filters can be applied to filter the result list.
+  is: [ secured ]
+  queryParameters:
+    cmd:
+      description: the command of this application
```

Comment:
"Filter the result to only return apps whose `cmd` field contains the given `cmd`, Default: `""`"